src/Microsoft.AspNetCore.Razor.Evolution/Legacy/CSharpCodeParser.cs

```diff
@@ -39,11 +39,16 @@ internal class CSharpCodeParser : TokenizerBackedParser<CSharpTokenizer, CSharpS
         private Dictionary<CSharpKeyword, Action<bool>> _keywordParsers = new Dictionary<CSharpKeyword, Action<bool>>();
 
         public CSharpCodeParser(ParserContext context)
+            : this (directiveDescriptors: Enumerable.Empty<DirectiveDescriptor>(), context: context)
```

Comment:
Should talk about how we want to replace existing directives with the directive descriptors. Aka:

1. Added in the CSharpCodeParser as "additional" directives?
2. Added in the person who constructs CSharpCodeParser in a manner that enables users to remove default directives from the system.